documentation/docs/05-misc/02-testing.md

```diff
@@ -2,7 +2,156 @@
 title: Testing
 ---
 
-- component testing basics
-- rune testing basics
-- vitest setup
-- e2e
+Testing helps you write and maintain your code and guard against regressions. Testing frameworks help you with that, allowing you to describe assertions or expectations about how your code should behave. Svelte is unopinionated about which testing framework you use, so you can write unit tests, integration tests, and end-to-end tests using solutions like Vitest, Jasmine, Cypress and Playwright.
+
+## Unit/Integration tests using Vitest
+
+Unit tests allow you to test small isolated parts of your code. Integration tests allow you to test parts of your application to see if they work together. If you're using Vite (including via SvelteKit), we recommend using [Vitest](https://vitest.dev/).
+
+To get started, install Vitest and JSDOM (a library that shims DOM APIs):
```

Comment:
jsdom refers to itself with lowercase letters
```suggestion
To get started, install Vitest and jsdom (a library that shims DOM APIs):
```

Reply:
Also could be worth also mentioning happy-dom?

Reply:
we've had issues with happy-dom in the past